pkg/instance/type.go

```diff
@@ -7,7 +7,6 @@ type InstanceType int
 const (
 	UnknownInstance InstanceType = iota
 	BasicInstance
-	ExecutorInstance
```

Comment:
This gave us the possibility to disable networking between Instances without disabling networking from and to executor instances. We would lose this ability.

The question is if we need the scenario where we disable the networking of a node for a certain time to simulate network connection loss.

Reply:
I see. Since we are removing executor from the instance package, we might want to define another type of instance which resists the network disabling tasks. Something like `AlwaysEnabledNetworkInstance` not very nice name :D but you get the point.

Reply:
If you agree with this idea, let's track it in a separate issue

Reply:
That makes sense 👍 We need to document that in a very clear way to the dev.

Reply:
an issue is created for it #522